class.jetpack.php

```diff
@@ -6160,7 +6160,16 @@ public static function validate_sync_error_idc_option() {
 			$local_options = self::get_sync_error_idc_option();
 			// Ensure all values are set.
 			if ( isset( $sync_error['home'] ) && isset( $local_options['home'] ) && isset( $sync_error['siteurl'] ) && isset( $local_options['siteurl'] ) ) {
-				if ( $sync_error['home'] === $local_options['home'] && $sync_error['siteurl'] === $local_options['siteurl'] ) {
+
+				// If the WP.com expected home and siteurl match local home and siteurl it is not valid IDC.
+				if (
```

Comment:
I like having this as an explicit check vs additional !== in the single if statement. However that is my preference so feel free to suggest otherwise :)